lib/tasks/copy_vetted_documents.rake

```diff
@@ -0,0 +1,38 @@
+desc 'Copy all vetted documents from source tenant to destination tenant'
+task :copy_vetted_documents, [:source_tenant, :destination_tenant]  => :environment do |t, args|
+
+  # set tenant variables
+  source = args[:source_tenant]
+  destination = args[:destination_tenant]
+  starting_tenant = Apartment::Tenant.current
+
+  # switch to source tenant
+  Apartment::Tenant.switch(source)
+
+  # gather all vetted documents from source tenant
+  vetted_docs = Document.where(vetted: true).to_a
```

Comment:
This could be hundreds or thousands of `Document` objects, each of which might have a large blob of text in the body field. Maybe we should pluck the ids, and iterate on those, so we're not storing a huge array in memory?

Reply:
This will involve doing the tenant switching within the loop, as I think about it. We should test with real data. You could copy the data from the main production tenant back to staging (probably just Documents, and a few other models and just for one tenant, excluding the tenants table, so you don't end up with inoperable tenants on staging), and see how it works on Heroku.  Let me know what you think!

Reply:
That's a good point. I tested this out on staging (after copying over data from the production db) with the tenant that had the largest number of documents (cove-studio, with 1951 total and 864 vetted), and there doesn't seem to be a performance issue (at least with the current number of documents) - the script ran successfully in under a minute.